### PR TITLE
chore(examples): add webiny cms example

### PR DESCRIPTION
**Pull request by endymion1818**
## Documentation / Examples

- [X] Make sure the linting passes by running `pnpm lint`
- [X] The "examples guidelines" are followed from [our contributing doc](https://github.com/vercel/next.js/blob/canary/contributing/examples/adding-examples.md)

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -400,22 +400,26 @@ importers:
       '@next/eslint-plugin-next': 12.3.2-canary.18
```
Comment (balazsorban44) [inline pnpm-lock.yaml:400]: The lock file should not be changed by examples, please revert the changes in this file.

File: examples/cms-webiny/.gitignore [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+yarn-error.log*
+
+# local env files
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+# vercel
+.vercel
```
Comment (balazsorban44) [inline examples/cms-webiny/.gitignore:34]:
```suggestion
.vercel

# typescript
*.tsbuildinfo
next-env.d.ts
```

File: examples/cms-webiny/pages/_document.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
```
Comment (balazsorban44) [inline examples/cms-webiny/pages/_document.tsx:1]: This file does not do anything (same as the default), let's remove it.

File: examples/cms-webiny/pages/posts/[slug].tsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
+import React from 'react'
```
Comment (balazsorban44) [inline examples/cms-webiny/pages/posts/[slug].tsx:1]:
```suggestion
```

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
Added some comments.

`React` imports are unnecessary unless you access a method like `React.useState` or types. In most cases, I could not see that, so please remove those imports from files (even if I did not request the change).

File: examples/cms-webiny/pages/_app.tsx [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import React from 'react'
```
[the remark this answers is not in the document]
Comment (endymion1818) [reply examples/cms-webiny/pages/_app.tsx:1]: I get type errors when I omit this

**Comment by endymion1818** [comment]
Thanks for your help @balazsorban44, the only thing I wasn't able to do was remove `import React from 'react'`, as it resulted in type errors.

**Comment by endymion1818** [comment]
Hey @balazsorban44, any other way around the react imports thing? I'd really like to get a clear way forward for this PR.

**Comment by balazsorban44** [comment]
You forgot to add a `tsconfig.json` file, so the IDE reported JSX as an error. I applied some suggestions and fixed up those issues.

File: examples/cms-webiny/tailwind.config.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
+module.exports = {
```
Comment (balazsorban44) [inline examples/cms-webiny/tailwind.config.js:1]:
```suggestion
/** @type {import('tailwindcss').Config} */
module.exports = {
```

**Review by balazsorban44 — APPROVED** [review]

**Comment by balazsorban44** [comment]
Thanks for the PR!

**Comment by balazsorban44** [comment]
Merged manually, since `pnpm-lock.yaml` was touched (then reverted) and I was not a CODEOWNER.

**Comment by endymion1818** [comment]
Thanks @balazsorban44
